docs/swagger/spec.yml

```diff
@@ -20,8 +20,12 @@ consumes:
 tags:
```

Comment:
Backend has the scene filter as sceneId, spec has it as scene for several endpoints (eg. footprints). I'm fine with either, but we should make an issue if the backend should change.